mission_control/navigator_missions/nav_missions/detect_deliver.py

```diff
@@ -181,13 +165,21 @@ def find_and_shoot(self):
         yield self.get_waypoint()  # Get waypoint of shooter target
         yield self.circle_search()  # Go to waypoint and circle until target found
         yield self.align_to_target()
-        yield self.offset_for_target()  # Move a little bit forward to be centered to target
         yield self.shoot_all_balls()
         yield self.navigator.vision_proxies["get_shape"].stop()
 
 @txros.util.cancellableInlineCallbacks
+def setup_mission(navigator):
+    color = yield navigator.mission_params["scan_the_code_color3"].get()
+    fprint("Setting search color to scan the code color 3 = {}".format(color), title="DETECT DELIVER",  msg_color='green')
```

Comment:
This isn't really a critical thing but I feel like it'd be good to enforce some standards with what to print in which color with fprint. Like green should be for things that could fail but succeed instead. I'd imagine these info prints would be blue? What are your thoughts